programs/client/Client.cpp

```diff
@@ -1188,6 +1188,9 @@ void Client::processConfig()
     /// - --queries-file command line option is present.
     ///   The value of the option is used as file with query (or of multiple queries) to execute.
 
+    if (config().has("query") && config().has("queries-file"))
```

Comment:
I think the comment in l. 1183ff applies to l. 1194. Could you move l. 1191/1192 before the comment?

Reply:
Done.